incubator/hnc/e2e/tests/e2e_test.go

```diff
@@ -38,3 +43,14 @@ func tryRun(cmdln ...string) error {
 	GinkgoT().Log("Output: ", string(stdout))
 	return err
 }
+
+func cleanupNamespaces(nses ...string) {
+	// Remove all possible objections HNC might have to deleting a network. Make sure it 
```

Comment:
s/network/namespace/

Reply:
what do you mean?

Reply:
Sorry - "s/FOO/BAR" is a `sed` command which means "find all text called FOO and replace it with BAR." It usually means "there's a typo, please fix it" - in this case, you wrote "... to deleting a **network**" when I'm pretty sure you mean "... to deleting a **namespace**"

Reply:
oh nice catch - and I learnt a new command